sbnobj/Common/Reco/CRUMBSResult.h

```diff
@@ -0,0 +1,105 @@
+// Class for holding the result of the CRUMBS slice ID
+
+#ifndef sbncode_CRUMBSResult_H
+#define sbncode_CRUMBSResult_H
+
+namespace sbn {
+  class CRUMBSResult {
+  public:
+
+    CRUMBSResult() 
+      : score(-5.f)
+      , tpc_CRFracHitsInLongestTrack(-5.f)
+      , tpc_CRLongestTrackDeflection(-5.f)
+      , tpc_CRLongestTrackDirY(-5.f)
+      , tpc_CRNHitsMax(-5.f)
+      , tpc_NuEigenRatioInSphere(-5.f)
+      , tpc_NuNFinalStatePfos(-5.f)
+      , tpc_NuNHitsTotal(-5.f)
+      , tpc_NuNSpacePointsInSphere(-5.f)
+      , tpc_NuVertexY(-5.f)
+      , tpc_NuWeightedDirZ(-5.f)
+      , tpc_StoppingChi2CosmicRatio(-5.f)
+      , pds_FMTotalScore(-5.f)
+      , pds_FMPE(-5.f)
+      , pds_FMTime(-5.f)
+      , crt_TrackScore(-5.f)
+      , crt_HitScore(-5.f)
+      , crt_TrackTime(-5.f)
+      , crt_HitTime(-5.f)
+    {
+    }
+    
+    CRUMBSResult(const float score)
+      : score(score)
+      , tpc_CRFracHitsInLongestTrack(-5.f)
+      , tpc_CRLongestTrackDeflection(-5.f)
+      , tpc_CRLongestTrackDirY(-5.f)
+      , tpc_CRNHitsMax(-5.f)
+      , tpc_NuEigenRatioInSphere(-5.f)
+      , tpc_NuNFinalStatePfos(-5.f)
+      , tpc_NuNHitsTotal(-5.f)
+      , tpc_NuNSpacePointsInSphere(-5.f)
+      , tpc_NuVertexY(-5.f)
+      , tpc_NuWeightedDirZ(-5.f)
+      , tpc_StoppingChi2CosmicRatio(-5.f)
+      , pds_FMTotalScore(-5.f)
+      , pds_FMPE(-5.f)
+      , pds_FMTime(-5.f)
+      , crt_TrackScore(-5.f)
+      , crt_HitScore(-5.f)
+      , crt_TrackTime(-5.f)
+      , crt_HitTime(-5.f)
+    {
+    }
+
+    CRUMBSResult(const float score, const float tpc_CRFracHitsInLongestTrack, const float tpc_CRLongestTrackDeflection, const float tpc_CRLongestTrackDirY, 
```

Comment:
Personally I don't think all these `consts` help in such a situation.

Reply:
Okay, will remove